### PR TITLE
Updated API-README

### PR DESCRIPTION
**Pull request by taratimmerman**
Description
**Invited Collaborator(s) to Review

Jennifer Kramer

What work was done?

Updated API-README.MD to include:
- New Recipient requirements
- New Household endpoints and requirements

Why was this work done?

This update was necessary as the API-README file contains information for the user about the endpoints and their requirements. and should stay up-to-date with any changes.

### CODE REVIEW

File: api/API-README.MD [hunk 1]
```diff
@@ -250,8 +250,8 @@ All routes work with /api/recipient or /api/recipients
 | ------------------ | ------ | ------------------------------------------------- | ---------------------------------------- |
 | /api/recipients    | GET    | none                                              | none                                     |
 | /api/recipient/:id | GET    | none                                              | none                                     |
-| /api/recipient     | POST   | name, age, city, state, zip_code, veteran_status, | admin, program_manager, service_provider |
-|                    |        | household_size                                    |                                          |
+| /api/recipient     | POST   | first_name, last_name, age, household_id          | admin, program_manager, service_provider |
+|                    |        | veteran_status                                    |                                          |
 | /api/recipient/:id | PUT    | desired edit as key:value                         | admin, program_manager                   |
 | /api/recipient/:id | DELETE | none                                              | admin                                    |
```
Comment (jenkrame21) [inline api/API-README.MD:257]: If you could, please change the returned object recipients examples to reflect the new changes, ie. name to first_name / last_name.

**Review by jenkrame21 — APPROVED** [review]
Other than my little comment, everything else looks perfect. Thank you! Let me know if you have any questions.

**Review by NRHietala — APPROVED** [review]
Reviewed with Jennifer on Zoom.

**Comment by taratimmerman** [comment]
I updated my changes with your suggestions and we should be good to go. Thank you for the thorough reviews, team!

**Pull request merged**